doc/api/deprecations.md

```diff
@@ -2644,7 +2644,7 @@ changes:
     description: Runtime deprecation.
 -->
 
-Type: Runtime
+Type: Documentation-only (supports [`--pending-deprecation`][])
```

Comment:
Is this changing the text for the wrong entry?

Reply:
Good catch! Thanks @addaleax!